src/content/docs/usage/workspaces.mdx

```diff
@@ -37,13 +37,19 @@ Creating a Workspace allows you to set up a new environment either by selecting
    daytona create
    ```
 
+
+2. Select the Git Provider and repository you want your Workspace to be configured with.
```

Comment:
We removed these sections from the workspaces page in https://github.com/daytonaio/docs/pull/96

Pleas only include the fixes to the links in this PR